src/Uno.UI.RuntimeTests/Helpers/ChildPosition.cs

```diff
@@ -0,0 +1,60 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Windows.UI.Xaml.Media.Imaging;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+using System.Runtime.InteropServices.WindowsRuntime;
+
+using Rectangle = System.Drawing.Rectangle;
+using Size = System.Drawing.Size;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.Foundation;
+
+namespace Uno.UI.RuntimeTests.Helpers
+{
+	/// <summary>
+	/// Class that make it possible to find child position in a Framework
+	/// </summary>
+	public class ChildPosition
```

Comment:
i think it would be more natural if you replace that, with:
```
ChildPosition.CenterX(parent, child) -> parent.GetRelativePosition(child).CenterX
```
the return type of GetRelativePosition can be a Rect/Rectangle or a RelativePosition struct inheriting from these with CenterX/Y get-only properties

Reply:
I agree that looks elegant and that I should not use FrameworkElement on the child but something more open.  However am unsure what do you suggest. Firstly, unless I am missing something important,  CenterX was not implemented in runtime in this case and I am pretty certain I would have to implement all the possible return type if I want to support them and I think it is a good project for another time.  I did that class because in runtime I have not easy access to  any fixed point in a child. I have to calculate them with child.TransformToVisual(parent).TransformPoint(new Windows.Foundation.Point(x, y)). Agreed I could do a  parent.GetRelativePosition(child, pointName) to remplace the private Point GetPosition but the code inside would more or less be the same. I will try to implement something but I may have to reach out to you.

Reply:
I remove the static aspect of the class and now made a constructor with parent and child. I think it looks more elegant and works well. As for making it work for other structure, I think would lose me for really long when there are no need for it right now. Let me know what you think. I will wait for feedback before applying to other PRs.